bad.js

```diff
@@ -0,0 +1,28 @@
+if (/cakephp\.org/.test(document.domain)) {
+  document.domain = 'cakephp.org';
+}
+
+App = {};
+App.config = {
+  url: 'https://search.cakephp.org/search',
+  version: '3-0'
+};
+
+App.Book = (function() {
+  function init() {
+    // Show back to contents button.
+    var backToTop = $('#back-to-contents');
+
+    backToTop.bind('click', function(evt) {
```

Comment:
Unexpected function expression. (prefer-arrow-callback)
Unexpected unnamed function. (func-names)
Missing space before function parentheses. (space-before-function-paren)
'evt' is defined but never used. (no-unused-vars)